src/tests/datastore-tests.ts

```diff
@@ -702,7 +729,7 @@ describe('postgres datastore', () => {
     expect([...addrDResult]).toEqual([]);
   });
 
-  test.only('pg block store and retrieve', async () => {
+  test('pg block store and retrieve', async () => {
```

Comment:
I think tests in this PR are broken because of this. This `test.only` slipped through a few months ago and since then some of these tests seem to have been broken.

Reply:
Yep, that is the issue https://github.com/hirosystems/stacks-blockchain-api/issues/1045